src/modules/ekf2/EKF/ekf.cpp

```diff
@@ -346,6 +347,38 @@ void Ekf::resetGlobalPosToExternalObservation(double lat_deg, double lon_deg, fl
 
 	resetHorizontalPositionToExternal(pos_corrected, math::max(accuracy, FLT_EPSILON));
 }
+void Ekf::resetWindToExternalObservation(float wind_speed, float wind_direction, float wind_speed_accuracy, float wind_direction_accuracy)
```

Comment:
Wind (including the full derivation update) can be entirely disabled at compile time.

```suggestion

#if defined(CONFIG_EKF2_WIND)
void Ekf::resetWindToExternalObservation(float wind_speed, float wind_direction, float wind_speed_accuracy, float wind_direction_accuracy)
```

Reply:
It might make sense to move all these misc wind pieces to a new file EKF/wind.cpp (like EKF/velocity_fusion.cpp, etc), then we could simply not include it in the build instead of including ifdefs.